acme/tests/crypto_util_test.py

```diff
@@ -223,6 +263,25 @@ def test_make_csr(self):
                 ).get_data(),
             )
 
+    def test_make_csr_ip(self):
+        csr_pem = self._call_with_key(["a.example", "127.0.0.1", "::1"])
+        self.assertIn(b'--BEGIN CERTIFICATE REQUEST--' , csr_pem)
+        self.assertIn(b'--END CERTIFICATE REQUEST--' , csr_pem)
+        csr = OpenSSL.crypto.load_certificate_request(
+            OpenSSL.crypto.FILETYPE_PEM, csr_pem)
+        # In pyopenssl 0.13 (used with TOXENV=py27-oldest), csr objects don't
+        # have a get_extensions() method, so we skip this test if the method
+        # isn't available.
+        if hasattr(csr, 'get_extensions'):
```

Comment:
A simple question: isn't this always true? pyopenssl 0.13 is quite old, and Python 2.7 is not supported by Certbot anymore.

Reply:
probably not, but other tests all have this so I think it's out of scope to remove it